src/config/themeConfig.js

```diff
@@ -23,82 +23,81 @@ import { primary as defaultPrimary } from "./themes/default/defaultTheme";
 import {
   primaryDarker as defaultPrimaryDarker,
   darkColor as defaultDarkColor,
-  headerColor as defaultBanner,
```

Comment:
@a-limyr I'm a little unsure about this change. There is no `headerColor` export in defaultTheme, so not sure where it came from. Nothing seems broken though

Reply:
I had the same issue. I think the Theme setup is broken as it is now, but I have made a new issue that should look at this as part of creating a common theme for the entire app. It works, but it is not totally clear why it does.